app/models/placement_suitability.rb

```diff
@@ -24,6 +25,17 @@ def sanitize_input
     self.address_line_2 = address_line_2.to_s.strip.presence if address_line_2
     self.address_city = address_city.to_s.strip.presence if address_city
     self.address_county = address_county.to_s.strip.presence if address_county
-    self.address_postcode = address_postcode.upcase.gsub(" ", "").presence if address_postcode
+  end
+
+  def sanitize_postcode
```

Comment:
We should probably extract the sanitising process and validation into a `PostcodeValidator`. That way it can be encapsulated, tested and just included in these 2 models where we need it. Perhaps can be done separately from this PR.

Reply:
I thought about that. The postcode_validator is an each_validator child...it starts to look a bit messy with multiple methods

Reply:
Also we need a conditional chain of validations, I think...

Reply:
Happy to look at that later 👍